webrtc-stats.html

```diff
@@ -1889,6 +1913,15 @@
                   or UDP. It is similar to the TIAS defined in [[!RFC3890]], i.e., it is measured
                   in bits per second and the bitrate is calculated over a 1 second window.
                 </p>
+                <p>
+                  Implementations that do not calculate a sender-side estimate should leave this undefined.
+                  Additionally, the value should be undefined for candidate pairs that were never used.
+                  For pairs in use, the estimate is normally no lower than the bitrate for
+                  the packets sent at <dfn>lastPacketSentTimestamp</dfn>, but might be higher.
+                  For candidate pairs that are not currently in use but were used before,
+                  implementations can use the last calculated estimate provided that no further
+                  packets are sent.
```

Comment:
All estimation algorithms start off at some initial value, so internally, there is some estimate (likely made with no real world input). It doesn't make much sense to expose this number.

It makes sense to me to only expose this number after we have some faith in it (which would be after sending media for a while), but I'm not sure how to phrase that.